docker/bookkeeper/entrypoint.sh

```diff
@@ -8,31 +8,32 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-set -e
-
-# To create directories for multiple ledgers and journals if specified
-create_dir() {
-  IFS=',' read -ra directories <<< $1
-  for i in "${directories[@]}"
-  do
-      mkdir -p $i
-      if [ "$(id -u)" = '0' ]; then
-          chown -R "${BK_USER}:${BK_USER}" $i
-      fi
-  done
-}
+# set -e
```

Comment:
Lets uncomment this line and things should work after the `sysctl` is properly fixed

Reply:
This change was a temporary one. It is resolved in one of the later commits that were pushed.